contrib/win32/openssh/OpenSSHTestHelper.psm1

```diff
@@ -309,26 +299,33 @@ function Get-UserSID
     Cleanup-OpenSSHTestEnvironment
 #>
 function Cleanup-OpenSSHTestEnvironment
-{    
+{   
+    if($Global:OpenSSHTestInfo -eq $null) {
+        throw "OpenSSHTestInfo is not set. Did you run Setup-OpenSShTestEnvironment?"
+    }
+
+    $sshBinPath = $Global:OpenSSHTestInfo["OpenSSHBinPath"]
+    $curdir = (pwd).Path
+
     # .exe - Windows specific. TODO - PAL 
-    if (-not (Test-Path (Join-Path $script:OpenSSHBinPath ssh.exe) -PathType Leaf))
+    if (-not (Test-Path (Join-Path $sshBinPath ssh.exe) -PathType Leaf))
     {
         Throw "Cannot find OpenSSH binaries under $script:OpenSSHBinPath. "
     }
 
+    cd $sshBinPath
     #unregister test host keys from agent
-    Get-ChildItem "$($script:OpenSSHBinPath)\sshtest*hostkey*.pub"| % {
-            $cmd = "cmd /c `"$env:ProgramData\chocolatey\lib\sysinternals\tools\psexec -accepteula -nobanner -s -w $($script:OpenSSHBinPath) ssh-add -d $_ 2> tmp.txt`""
-            iex $cmd
+    Get-ChildItem "sshtest*hostkey*.pub"| % {
```

Comment:
if you use "$sshBinPath\sshtest*hostkey*.pub", you don't need to jump between directories any more.  The downside of jumping directory is that if anything goes wrong between the code, you won't come back.